src/OSPSuite.InstallationValidator.Core/Assets/Captions.cs

```diff
@@ -70,10 +66,15 @@ public static string InColor(string stringToFormat, Color color)
          return $"<span style='color:rgb({color.R},{color.G},{color.B})'>{stringToFormat}</span>";
       }
 
+      public static string ReportCreatedUnder(string reportConfigurationReportFile)
+      {
+         return $"Installation report was created and saved under {InBold(reportConfigurationReportFile)}";
```

Comment:
Could say it was 'created and saved as' if the bold text will be the full output path including file name